quantum/rgb_matrix/rgb_matrix.h

```diff
@@ -213,6 +232,17 @@ typedef struct {
     void (*flush)(void);
 } rgb_matrix_driver_t;
 
+#if defined(RGB_MATRIX_SPLIT)
+#    define RGB_MATRIX_FINISHED_ALL_LEDS            \
```

Comment:
This might be better as a function rather than a define, to keep filesize down (especially on AVR...)

Reply:
Couple of potential issues. One is that the function definition changes depending on the define (it may or may not need k_rgb_matrix_split). Not sure how to go around that. 
Second issue is that I don't know if inlining would work, if performance (over file size) is desired.

Reply:
I'm pretty sure something like this would work fine:
```c
bool rgb_matrix_finished_all_leds(void) {
#if defined(RGB_MATRIX_SPLIT)
    if (is_keyboard_left()) {
        return led_max < k_rgb_matrix_split[0];
    } else {
        return led_max < DRIVER_LED_TOTAL;
    }
#else
    return led_max < DRIVER_LED_TOTAL;
#endif
}
```

Reply:
Updated.